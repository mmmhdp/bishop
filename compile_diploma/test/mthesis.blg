[0] Config.pm:307> INFO - This is Biber 2.19
[0] Config.pm:310> INFO - Logfile is 'mthesis.blg'
[95] biber:340> INFO - === Thu May 29, 2025, 10:43:07
[118] Biber.pm:419> INFO - Reading 'mthesis.bcf'
[233] Biber.pm:979> INFO - Found 26 citekeys in bib section 0
[252] Biber.pm:4419> INFO - Processing section 0
[265] Biber.pm:4610> INFO - Looking for bibtex file 'bibliography.bib' for section 0
[268] bibtex.pm:1713> INFO - LaTeX decoding ...
[306] bibtex.pm:1519> INFO - Found BibTeX data source 'bibliography.bib'
[871] UCollate.pm:68> INFO - Overriding locale 'ru-RU' defaults 'variable = shifted' with 'variable = non-ignorable'
[871] UCollate.pm:68> INFO - Overriding locale 'ru-RU' defaults 'normalization = NFD' with 'normalization = prenormalized'
[871] Biber.pm:4239> INFO - Sorting list 'none/global//global/global' of type 'entry' with template 'none' and locale 'ru-RU'
[871] Biber.pm:4245> INFO - No sort tailoring available for locale 'ru-RU'
[909] bbl.pm:660> INFO - Writing 'mthesis.bbl' with encoding 'UTF-8'
[932] bbl.pm:763> INFO - Output to mthesis.bbl
