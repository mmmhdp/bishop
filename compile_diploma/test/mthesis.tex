\documentclass[
    % master,
    colurls  
]{mthesis}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Параметры класса документа

%---------
% тип ВКР
%---------
% По умолчанию тип ВКР установлен как "bachelor", но может быть заменен 
% на "master" для магистерской ВКР.

%-------------
% гиперссылки
%-------------
% blackurls - ссылки на литературу, таблицы, рисунки, формулы и т.п.
%             не выделяются цветом (используется для ч/б печати).
% colurls   - цветные ссылки (для распространения рукописи в эл. виде).


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Здесь следует размещать допольнительные пакеты и команды. Следующие 
% пакеты уже загружены в стилевой файл: geometry, amssymb, mathtools, 
% ifthen, cmap, textcomp, babel, upgreek, indentfirst, xcolor, 
% tabularray, icomma, ulem, soulutf8, tikz, hyperref, graphicx, 
% caption, subcaption, pdfpages, aliascnt, totalcount, totcount, 
% totpages, placeins, enumitem, biblatex, fancyvrb, listings

% например, для создания данного примера ВКР потребовались 
% следующие пакеты и команды:
\usepackage{graphicx}  
\usepackage{listings}
\usepackage{xcolor}
\definecolor{codepurple}{rgb}{0.58,0,0.82}
\definecolor{codegray}{rgb}{0.5,0.5,0.5}

\lstdefinestyle{jsonstyle}{
    backgroundcolor=\color{white},
    commentstyle=\color{codegreen},
    keywordstyle=\color{magenta},
    numberstyle=\tiny\color{codegray},
    stringstyle=\color{codepurple},
    basicstyle=\ttfamily\footnotesize,
    breakatwhitespace=false,
    breaklines=true,
    captionpos=b,
    keepspaces=true,
    numbers=left,
    numbersep=5pt,
    showspaces=false,
    showstringspaces=false,
    showtabs=false,
    tabsize=2,
    frame=single
}
% пакет для вёрстки сложных матриц
\usepackage{nicematrix}

% пакет для вставки примеров изображений
\usepackage{duckuments}

% пакет для оформления блоков цитирования
\usepackage{csquotes}

% доп. библиотеки tikz
\usetikzlibrary{shapes.geometric, shapes.misc, arrows, positioning}

% стили tikz для рисования блок-схемы
\tikzset{
  line/.style = {thick},
  startstop/.style = {rounded rectangle, line, minimum width=3cm, 
    minimum height=1cm, text centered, draw=black, fill=white},
  io/.style = {trapezium, line, trapezium stretches=true, trapezium left angle=70, 
    trapezium right angle=110, minimum width=3cm, minimum height=1cm, 
    text centered, draw=black, fill=white},
  process/.style = {rectangle, line, minimum width=3cm, minimum height=1cm, 
    text centered, text width=3cm, draw=black, fill=white},
  decision/.style = {diamond, line, minimum width=3cm, minimum height=1cm, 
    text centered, draw=black, fill=white, aspect=2},
  arrow/.style = {line,->,>=stealth},
}



\input{title.cfg}

\begin{document}

% Реферат
\include{abstract}

% Содержание
\ifdefmacro{\microtypesetup}{\microtypesetup{protrusion=false}}{}
\tableofcontents*
\ifdefmacro{\microtypesetup}{\microtypesetup{protrusion=true}}{}

% Список сокращений и условных обозначений
\include{acronyms}

% Введение
\include{introduction}

% Разделы ВКР
\include{02_структурный и системный анализ исследуемого объекта}
\include{03_обзор научно-технических источников}
\include{04_постановка задачи}
\include{05_сущность решения задачи}
\include{06_построение модели решения задачи}
\include{07_программная реализация}
\include{08_результаты}

% Заключение
\include{conclusion}

% Список литературы
\clearpage
%\hypersetup{ urlcolor=black }               % Ссылки делаем чёрными
%\providecommand*{\BibDash}{}                % В стилях ugost2008 отключаем использование тире как разделителя
\urlstyle{rm}                               % ссылки URL обычным шрифтом
\ifdefmacro{\microtypesetup}{\microtypesetup{protrusion=false}}{}
\insertbibliofull
\ifdefmacro{\microtypesetup}{\microtypesetup{protrusion=true}}{}
\urlstyle{tt}                               % возвращаем установки шрифта ссылок URL
%\hypersetup{ urlcolor={urlcolor} }          % Восстанавливаем цвет ссылок

% Настройка приложений
\ThesisAppendix

% Приложения
\include{appendix}

\end{document}
