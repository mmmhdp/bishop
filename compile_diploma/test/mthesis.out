\BOOKMARK [0][-]{Referat.0}{\376\377\004\020\004\035\004\035\004\036\004\042\004\020\004\046\004\030\004\057}{}% 1
\BOOKMARK [0][-]{Abstract.0}{\376\377\000A\000B\000S\000T\000R\000A\000C\000T}{}% 2
\BOOKMARK [0][-]{chapter*.4}{\376\377\004\037\004\025\004\040\004\025\004\047\004\025\004\035\004\054\000\040\004\041\004\036\004\032\004\040\004\020\004\051\004\025\004\035\004\030\004\031\000\040\004\030\000\040\004\036\004\021\004\036\004\027\004\035\004\020\004\047\004\025\004\035\004\030\004\031}{}% 3
\BOOKMARK [0][-]{chapter*.5}{\376\377\004\022\004\022\004\025\004\024\004\025\004\035\004\030\004\025}{}% 4
\BOOKMARK [0][-]{chapter.1}{\376\377\004\041\004\102\004\100\004\103\004\072\004\102\004\103\004\100\004\075\004\113\004\071\000\040\004\070\000\040\004\101\004\070\004\101\004\102\004\065\004\074\004\075\004\113\004\071\000\040\004\060\004\075\004\060\004\073\004\070\004\067\000\040\004\070\004\101\004\101\004\073\004\065\004\064\004\103\004\065\004\074\004\076\004\063\004\076\000\040\004\076\004\061\004\112\004\065\004\072\004\102\004\060}{}% 5
\BOOKMARK [0][-]{chapter.2}{\376\377\004\036\004\061\004\067\004\076\004\100\000\040\004\075\004\060\004\103\004\107\004\075\004\076\000-\004\102\004\065\004\105\004\075\004\070\004\107\004\065\004\101\004\072\004\070\004\105\000\040\004\070\004\101\004\102\004\076\004\107\004\075\004\070\004\072\004\076\004\062\000\040\004\070\004\075\004\104\004\076\004\100\004\074\004\060\004\106\004\070\004\070}{}% 6
\BOOKMARK [1][-]{section.2.1}{\376\377\004\032\004\076\004\075\004\106\004\065\004\077\004\102\004\103\004\060\004\073\004\114\004\075\004\113\004\065\000\040\004\070\000\040\004\060\004\100\004\105\004\070\004\102\004\065\004\072\004\102\004\103\004\100\004\075\004\113\004\065\000\040\004\100\004\065\004\110\004\065\004\075\004\070\004\117}{chapter.2}% 7
\BOOKMARK [2][-]{subsection.2.1.1}{\376\377\004\036\004\061\004\111\004\070\004\071\000\040\004\060\004\100\004\105\004\070\004\102\004\065\004\072\004\102\004\103\004\100\004\075\004\113\004\071\000\040\004\077\004\060\004\102\004\102\004\065\004\100\004\075\000.\000\040\004\034\004\070\004\072\004\100\004\076\004\101\004\065\004\100\004\062\004\070\004\101\004\075\004\113\004\071\000\040\004\070\000\040\004\074\004\076\004\075\004\076\004\073\004\070\004\102\004\075\004\113\004\071\000\040\004\077\004\076\004\064\004\105\004\076\004\064\004\113}{section.2.1}% 8
\BOOKMARK [2][-]{subsection.2.1.2}{\376\377\004\037\004\060\004\102\004\102\004\065\004\100\004\075\000\040\004\062\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\117\000\040\004\074\004\070\004\072\004\100\004\076\004\101\004\065\004\100\004\062\004\070\004\101\004\076\004\062\000.\000\040\004\045\004\076\004\100\004\065\004\076\004\063\004\100\004\060\004\104\004\070\004\117\000\040\004\070\000\040\004\076\004\100\004\072\004\065\004\101\004\102\004\100\004\060\004\106\004\070\004\117}{section.2.1}% 9
\BOOKMARK [2][-]{subsection.2.1.3}{\376\377\004\034\004\076\004\064\004\065\004\073\004\114\000\040\004\062\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\117\000\040\004\072\004\076\004\074\004\077\004\076\004\075\004\065\004\075\004\102\004\076\004\062\000.\000\040\004\020\004\101\004\070\004\075\004\105\004\100\004\076\004\075\004\075\004\076\004\101\004\102\004\114\000,\000\040\004\077\004\060\004\100\004\060\004\073\004\073\004\065\004\073\004\114\004\075\004\076\004\101\004\102\004\114\000\040\004\070\000\040\004\072\004\076\004\075\004\072\004\103\004\100\004\065\004\075\004\102\004\075\004\076\004\101\004\102\004\114}{section.2.1}% 10
\BOOKMARK [2][-]{subsection.2.1.4}{\376\377\004\034\004\076\004\064\004\065\004\073\004\114\000\040\004\105\004\100\004\060\004\075\004\065\004\075\004\070\004\117\000\040\004\064\004\060\004\075\004\075\004\113\004\105}{section.2.1}% 11
\BOOKMARK [2][-]{subsection.2.1.5}{\376\377\004\037\004\060\004\102\004\102\004\065\004\100\004\075\000\040\004\072\004\073\004\070\004\065\004\075\004\102\004\101\004\072\004\076\004\071\000\040\004\060\004\100\004\105\004\070\004\102\004\065\004\072\004\102\004\103\004\100\004\113\000.\000\040\004\034\004\076\004\064\004\065\004\073\004\070\000\040\000M\000P\000A\000\040\004\070\000\040\000S\000P\000A}{section.2.1}% 12
\BOOKMARK [1][-]{section.2.2}{\376\377\004\037\004\100\004\076\004\063\004\100\004\060\004\074\004\074\004\075\004\113\004\065\000\040\004\101\004\100\004\065\004\064\004\101\004\102\004\062\004\060\000\040\004\070\000\040\004\102\004\065\004\105\004\075\004\076\004\073\004\076\004\063\004\070\004\070\000\040\004\100\004\065\004\060\004\073\004\070\004\067\004\060\004\106\004\070\004\070}{chapter.2}% 13
\BOOKMARK [2][-]{subsection.2.2.1}{\376\377\004\022\004\113\004\061\004\076\004\100\000\040\004\117\004\067\004\113\004\072\004\060\000\040\004\077\004\100\004\076\004\063\004\100\004\060\004\074\004\074\004\070\004\100\004\076\004\062\004\060\004\075\004\070\004\117}{section.2.2}% 14
\BOOKMARK [2][-]{subsection.2.2.2}{\376\377\004\022\004\065\004\061\000\040\004\070\004\075\004\102\004\065\004\100\004\104\004\065\004\071\004\101}{section.2.2}% 15
\BOOKMARK [2][-]{subsection.2.2.3}{\376\377\004\041\004\065\004\100\004\062\004\065\004\100\004\075\004\060\004\117\000\040\004\107\004\060\004\101\004\102\004\114\000.\000\040\004\036\004\100\004\072\004\065\004\101\004\102\004\100\004\060\004\102\004\076\004\100}{section.2.2}% 16
\BOOKMARK [2][-]{subsection.2.2.4}{\376\377\004\041\004\065\004\100\004\062\004\065\004\100\004\075\004\060\004\117\000\040\004\107\004\060\004\101\004\102\004\114\000.\000\040\004\034\004\076\004\064\004\103\004\073\004\114\000\040\004\100\004\060\004\061\004\076\004\102\004\113\000\040\004\101\000\040\004\117\004\067\004\113\004\072\004\076\004\062\004\076\004\071\000\040\004\070\000\040\004\067\004\062\004\103\004\072\004\076\004\062\004\076\004\071\000\040\004\074\004\076\004\064\004\065\004\073\004\114\004\116}{section.2.2}% 17
\BOOKMARK [2][-]{subsection.2.2.5}{\376\377\004\041\004\065\004\100\004\062\004\065\004\100\004\075\004\060\004\117\000\040\004\107\004\060\004\101\004\102\004\114\000.\000\040\004\042\004\065\004\105\004\075\004\076\004\073\004\076\004\063\004\070\004\117\000\040\004\100\004\060\004\101\004\077\004\076\004\067\004\075\004\060\004\062\004\060\004\075\004\070\004\117\000\040\004\100\004\065\004\107\004\070}{section.2.2}% 18
\BOOKMARK [2][-]{subsection.2.2.6}{\376\377\004\041\004\065\004\100\004\062\004\065\004\100\004\075\004\060\004\117\000\040\004\107\004\060\004\101\004\102\004\114\000.\000\040\004\021\004\100\004\076\004\072\004\065\004\100\000\040\004\101\004\076\004\076\004\061\004\111\004\065\004\075\004\070\004\071}{section.2.2}% 19
\BOOKMARK [2][-]{subsection.2.2.7}{\376\377\004\030\004\075\004\104\004\100\004\060\004\101\004\102\004\100\004\103\004\072\004\102\004\103\004\100\004\060\000\040\004\070\000\040\004\100\004\060\004\067\004\062\004\065\004\100\004\102\004\113\004\062\004\060\004\075\004\070\004\065}{section.2.2}% 20
\BOOKMARK [2][-]{subsection.2.2.8}{\376\377\004\030\004\102\004\076\004\063\000\040\004\077\004\076\000\040\004\060\004\100\004\105\004\070\004\102\004\065\004\072\004\102\004\103\004\100\004\065}{section.2.2}% 21
\BOOKMARK [0][-]{chapter.3}{\376\377\004\037\004\076\004\101\004\102\004\060\004\075\004\076\004\062\004\072\004\060\000\040\004\067\004\060\004\064\004\060\004\107\004\070}{}% 22
\BOOKMARK [0][-]{chapter.4}{\376\377\004\041\004\103\004\111\004\075\004\076\004\101\004\102\004\114\000\040\004\100\004\065\004\110\004\065\004\075\004\070\004\117\000\040\004\067\004\060\004\064\004\060\004\107\004\070}{}% 23
\BOOKMARK [1][-]{section.4.1}{\376\377\004\032\004\076\004\074\004\077\004\076\004\075\004\065\004\075\004\102\004\113\000\040\004\100\004\065\004\110\004\065\004\075\004\070\004\117}{chapter.4}% 24
\BOOKMARK [2][-]{subsection.4.1.1}{\376\377\004\022\004\065\004\061\000\040\004\070\004\075\004\102\004\065\004\100\004\104\004\065\004\071\004\101}{section.4.1}% 25
\BOOKMARK [2][-]{subsection.4.1.2}{\376\377\004\021\004\065\004\072\004\115\004\075\004\064}{section.4.1}% 26
\BOOKMARK [2][-]{subsection.4.1.3}{\376\377\004\040\004\065\004\073\004\117\004\106\004\070\004\076\004\075\004\075\004\060\004\117\000\040\004\061\004\060\004\067\004\060\000\040\004\064\004\060\004\075\004\075\004\113\004\105}{section.4.1}% 27
\BOOKMARK [2][-]{subsection.4.1.4}{\376\377\000M\000L\000\040\004\101\004\065\004\100\004\062\004\070\004\101}{section.4.1}% 28
\BOOKMARK [2][-]{subsection.4.1.5}{\376\377\004\021\004\100\004\076\004\072\004\065\004\100\000\040\004\101\004\076\004\076\004\061\004\111\004\065\004\075\004\070\004\071}{section.4.1}% 29
\BOOKMARK [2][-]{subsection.4.1.6}{\376\377\000S\0003\000\040\004\105\004\100\004\060\004\075\004\070\004\073\004\070\004\111\004\065}{section.4.1}% 30
\BOOKMARK [1][-]{section.4.2}{\376\377\004\020\004\073\004\063\004\076\004\100\004\070\004\102\004\074\000\040\004\062\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\117}{chapter.4}% 31
\BOOKMARK [2][-]{subsection.4.2.1}{\376\377\004\037\004\100\004\076\004\106\004\065\004\064\004\103\004\100\004\113\000\040\004\100\004\065\004\063\004\070\004\101\004\102\004\100\004\060\004\106\004\070\004\070\000\040\004\070\000\040\004\060\004\103\004\102\004\065\004\075\004\102\004\070\004\104\004\070\004\072\004\060\004\106\004\070\004\070\000\040\004\077\004\076\004\073\004\114\004\067\004\076\004\062\004\060\004\102\004\065\004\073\004\117}{section.4.2}% 32
\BOOKMARK [2][-]{subsection.4.2.2}{\376\377\004\037\004\100\004\076\004\106\004\065\004\064\004\103\004\100\004\060\000\040\004\076\004\061\004\103\004\107\004\065\004\075\004\070\004\117}{section.4.2}% 33
\BOOKMARK [2][-]{subsection.4.2.3}{\376\377\004\037\004\100\004\076\004\106\004\065\004\064\004\103\004\100\004\060\000\040\004\062\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\117\000\040\004\101\000\040\004\060\004\062\004\060\004\102\004\060\004\100\004\076\004\074}{section.4.2}% 34
\BOOKMARK [0][-]{chapter.5}{\376\377\004\037\004\076\004\101\004\102\004\100\004\076\004\065\004\075\004\070\004\065\000\040\004\074\004\076\004\064\004\065\004\073\004\070\000\040\004\100\004\065\004\110\004\065\004\075\004\070\004\117\000\040\004\067\004\060\004\064\004\060\004\107\004\070}{}% 35
\BOOKMARK [1][-]{section.5.1}{\376\377\004\046\004\065\004\073\004\070\000\040\004\070\000\040\004\076\004\063\004\100\004\060\004\075\004\070\004\107\004\065\004\075\004\070\004\117}{chapter.5}% 36
\BOOKMARK [1][-]{section.5.2}{\376\377\004\034\004\076\004\064\004\103\004\073\004\114\000\040\004\063\004\065\004\075\004\065\004\100\004\060\004\106\004\070\004\070\000\040\004\102\004\065\004\072\004\101\004\102\004\060}{chapter.5}% 37
\BOOKMARK [2][-]{subsection.5.2.1}{\376\377\004\024\004\060\004\075\004\075\004\113\004\065}{section.5.2}% 38
\BOOKMARK [2][-]{subsection.5.2.2}{\376\377\004\022\004\113\004\061\004\076\004\100\000\040\004\061\004\060\004\067\004\076\004\062\004\076\004\071\000\040\004\074\004\076\004\064\004\065\004\073\004\070}{section.5.2}% 39
\BOOKMARK [2][-]{subsection.5.2.3}{\376\377\004\034\004\065\004\102\004\100\004\070\004\072\004\070}{section.5.2}% 40
\BOOKMARK [2][-]{subsection.5.2.4}{\376\377\004\042\004\076\004\072\004\065\004\075\004\070\004\067\004\060\004\106\004\070\004\117}{section.5.2}% 41
\BOOKMARK [2][-]{subsection.5.2.5}{\376\377\004\036\004\061\004\103\004\107\004\065\004\075\004\070\004\065}{section.5.2}% 42
\BOOKMARK [2][-]{subsection.5.2.6}{\376\377\004\023\004\065\004\075\004\065\004\100\004\060\004\106\004\070\004\117\000\040\004\102\004\065\004\072\004\101\004\102\004\060}{section.5.2}% 43
\BOOKMARK [1][-]{section.5.3}{\376\377\004\034\004\076\004\064\004\103\004\073\004\114\000\040\004\063\004\065\004\075\004\065\004\100\004\060\004\106\004\070\004\070\000\040\004\067\004\062\004\103\004\072\004\060}{chapter.5}% 44
\BOOKMARK [2][-]{subsection.5.3.1}{\376\377\004\022\004\113\004\061\004\076\004\100\000\040\004\061\004\060\004\067\004\076\004\062\004\076\004\071\000\040\004\074\004\076\004\064\004\065\004\073\004\070}{section.5.3}% 45
\BOOKMARK [2][-]{subsection.5.3.2}{\376\377\004\037\004\076\004\064\004\063\004\076\004\102\004\076\004\062\004\072\004\060\000\040\004\064\004\060\004\075\004\075\004\113\004\105}{section.5.3}% 46
\BOOKMARK [2][-]{subsection.5.3.3}{\376\377\004\023\004\070\004\077\004\065\004\100\004\077\004\060\004\100\004\060\004\074\004\065\004\102\004\100\004\113\000\040\004\064\004\076\004\076\004\061\004\103\004\107\004\065\004\075\004\070\004\117\000\040\004\074\004\076\004\064\004\065\004\073\004\070}{section.5.3}% 47
\BOOKMARK [2][-]{subsection.5.3.4}{\376\377\004\037\004\100\004\076\004\106\004\065\004\101\004\101\000\040\004\064\004\076\004\076\004\061\004\103\004\107\004\065\004\075\004\070\004\117\000\040\004\070\000\040\004\076\004\102\004\101\004\073\004\065\004\066\004\070\004\062\004\060\004\075\004\070\004\065\000\040\004\077\004\060\004\100\004\060\004\074\004\065\004\102\004\100\004\076\004\062}{section.5.3}% 48
\BOOKMARK [2][-]{subsection.5.3.5}{\376\377\004\036\004\106\004\065\004\075\004\072\004\060\000\040\004\072\004\060\004\107\004\065\004\101\004\102\004\062\004\060\000\040\004\063\004\065\004\075\004\065\004\100\004\060\004\106\004\070\004\070\000\040\004\070\000\040\004\070\004\101\004\077\004\076\004\073\004\114\004\067\004\103\004\065\004\074\004\113\004\065\000\040\004\074\004\065\004\102\004\100\004\070\004\072\004\070}{section.5.3}% 49
\BOOKMARK [0][-]{chapter.6}{\376\377\004\037\004\100\004\076\004\063\004\100\004\060\004\074\004\074\004\075\004\060\004\117\000\040\004\100\004\065\004\060\004\073\004\070\004\067\004\060\004\106\004\070\004\117}{}% 50
\BOOKMARK [1][-]{section.6.1}{\376\377\004\022\004\065\004\061\000-\004\070\004\075\004\102\004\065\004\100\004\104\004\065\004\071\004\101}{chapter.6}% 51
\BOOKMARK [2][-]{subsection.6.1.1}{\376\377\004\023\004\076\004\101\004\102\004\065\004\062\004\060\004\117\000\040\004\101\004\102\004\100\004\060\004\075\004\070\004\106\004\060}{section.6.1}% 52
\BOOKMARK [2][-]{subsection.6.1.2}{\376\377\004\030\004\064\004\065\004\075\004\102\004\070\004\104\004\070\004\072\004\060\004\106\004\070\004\117\000,\000\040\004\060\004\103\004\102\004\065\004\075\004\102\004\070\004\104\004\070\004\072\004\060\004\106\004\070\004\117\000\040\004\070\000\040\004\060\004\062\004\102\004\076\004\100\004\070\004\067\004\060\004\106\004\070\004\117}{section.6.1}% 53
\BOOKMARK [2][-]{subsection.6.1.3}{\376\377\004\036\004\101\004\075\004\076\004\062\004\075\004\060\004\117\000\040\004\101\004\102\004\100\004\060\004\075\004\070\004\106\004\060\000\040\004\077\004\100\004\070\004\073\004\076\004\066\004\065\004\075\004\070\004\117}{section.6.1}% 54
\BOOKMARK [2][-]{subsection.6.1.4}{\376\377\004\041\004\102\004\100\004\060\004\075\004\070\004\106\004\060\000\040\004\100\004\060\004\061\004\076\004\102\004\113\000\040\004\101\000\040\004\060\004\062\004\060\004\102\004\060\004\100\004\076\004\074}{section.6.1}% 55
\BOOKMARK [2][-]{subsection.6.1.5}{\376\377\004\030\004\075\004\102\004\065\004\100\004\104\004\065\004\071\004\101\000\040\004\076\004\061\004\111\004\065\004\075\004\070\004\117\000\040\004\101\000\040\004\060\004\062\004\060\004\102\004\060\004\100\004\076\004\074}{section.6.1}% 56
\BOOKMARK [2][-]{subsection.6.1.6}{\376\377\004\030\004\075\004\102\004\065\004\100\004\104\004\065\004\071\004\101\000\040\004\076\004\061\004\103\004\107\004\065\004\075\004\070\004\117\000\040\004\060\004\062\004\060\004\102\004\060\004\100\004\060}{section.6.1}% 57
\BOOKMARK [1][-]{section.6.2}{\376\377\000B\000a\000c\000k\000e\000n\000d\000-\004\101\004\065\004\100\004\062\004\070\004\101}{chapter.6}% 58
\BOOKMARK [2][-]{subsection.6.2.1}{\376\377\004\022\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\065\000\040\004\101\000\040\004\062\004\065\004\061\000-\004\070\004\075\004\102\004\065\004\100\004\104\004\065\004\071\004\101\004\076\004\074}{section.6.2}% 59
\BOOKMARK [2][-]{subsection.6.2.2}{\376\377\004\022\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\065\000\040\004\101\000\040\004\061\004\060\004\067\004\076\004\071\000\040\004\064\004\060\004\075\004\075\004\113\004\105}{section.6.2}% 60
\BOOKMARK [2][-]{subsection.6.2.3}{\376\377\004\022\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\065\000\040\004\101\000\040\000M\000L\000-\004\101\004\065\004\100\004\062\004\070\004\101\004\076\004\074}{section.6.2}% 61
\BOOKMARK [2][-]{subsection.6.2.4}{\376\377\004\022\004\067\004\060\004\070\004\074\004\076\004\064\004\065\004\071\004\101\004\102\004\062\004\070\004\065\000\040\004\101\000\040\000S\0003\000-\004\105\004\100\004\060\004\075\004\070\004\073\004\070\004\111\004\065\004\074}{section.6.2}% 62
\BOOKMARK [1][-]{section.6.3}{\376\377\004\041\004\065\004\100\004\062\004\070\004\101\000\040\004\063\004\065\004\075\004\065\004\100\004\060\004\106\004\070\004\070\000\040\004\067\004\062\004\103\004\072\004\060}{chapter.6}% 63
\BOOKMARK [2][-]{subsection.6.3.1}{\376\377\004\036\004\061\004\100\004\060\004\061\004\076\004\102\004\072\004\060\000\040\004\101\004\076\004\061\004\113\004\102\004\070\004\071\000\040\004\061\004\100\004\076\004\072\004\065\004\100\004\060\000\040\004\101\004\076\004\076\004\061\004\111\004\065\004\075\004\070\004\071\000\040\004\070\000\040\004\070\004\075\004\070\004\106\004\070\004\060\004\073\004\070\004\067\004\060\004\106\004\070\004\117\000\040\004\076\004\107\004\065\004\100\004\065\004\064\004\070\000\040\004\063\004\065\004\075\004\065\004\100\004\060\004\106\004\070\004\070}{section.6.3}% 64
\BOOKMARK [2][-]{subsection.6.3.2}{\376\377\004\055\004\102\004\060\004\077\004\113\000\040\004\101\004\070\004\075\004\102\004\065\004\067\004\060\000\040\004\100\004\065\004\107\004\070}{section.6.3}% 65
\BOOKMARK [2][-]{subsection.6.3.3}{\376\377\004\044\004\076\004\100\004\074\004\060\004\102\000\040\004\076\004\061\004\100\004\060\004\102\004\075\004\076\004\071\000\040\004\101\004\062\004\117\004\067\004\070\000\040\004\107\004\065\004\100\004\065\004\067\000\040\004\061\004\100\004\076\004\072\004\065\004\100\000\040\004\101\004\076\004\076\004\061\004\111\004\065\004\075\004\070\004\071}{section.6.3}% 66
\BOOKMARK [1][-]{section.6.4}{\376\377\004\041\004\065\004\100\004\062\004\070\004\101\000\040\004\063\004\065\004\075\004\065\004\100\004\060\004\106\004\070\004\070\000\040\004\102\004\065\004\072\004\101\004\102\004\060}{chapter.6}% 67
\BOOKMARK [2][-]{subsection.6.4.1}{\376\377\004\037\004\060\004\071\004\077\004\073\004\060\004\071\004\075\000\040\004\076\004\061\004\103\004\107\004\065\004\075\004\070\004\117\000\040\004\074\004\076\004\064\004\065\004\073\004\070}{section.6.4}% 68
\BOOKMARK [0][-]{chapter.7}{\376\377\004\040\004\065\004\067\004\103\004\073\004\114\004\102\004\060\004\102\004\113}{}% 69
\BOOKMARK [1][-]{section.7.1}{\376\377\004\030\004\075\004\070\004\106\004\070\004\060\004\073\004\070\004\067\004\060\004\106\004\070\004\117\000\040\004\101\004\070\004\101\004\102\004\065\004\074\004\113}{chapter.7}% 70
\BOOKMARK [1][-]{section.7.2}{\376\377\004\040\004\065\004\063\004\070\004\101\004\102\004\100\004\060\004\106\004\070\004\117\000\040\004\070\000\040\004\101\004\076\004\067\004\064\004\060\004\075\004\070\004\065\000\040\004\060\004\062\004\060\004\102\004\060\004\100\004\060}{chapter.7}% 71
\BOOKMARK [1][-]{section.7.3}{\376\377\004\027\004\060\004\063\004\100\004\103\004\067\004\072\004\060\000\040\004\076\004\061\004\103\004\107\004\060\004\116\004\111\004\070\004\105\000\040\004\074\004\060\004\102\004\065\004\100\004\070\004\060\004\073\004\076\004\062}{chapter.7}% 72
\BOOKMARK [1][-]{section.7.4}{\376\377\004\036\004\061\004\103\004\107\004\065\004\075\004\070\004\065\000\040\004\060\004\062\004\060\004\102\004\060\004\100\004\060}{chapter.7}% 73
\BOOKMARK [1][-]{section.7.5}{\376\377\004\024\004\070\004\060\004\073\004\076\004\063\000\040\004\070\000\040\004\101\004\070\004\075\004\102\004\065\004\067\000\040\004\100\004\065\004\107\004\070}{chapter.7}% 74
\BOOKMARK [0][-]{chapter*.36}{\376\377\004\027\004\020\004\032\004\033\004\056\004\047\004\025\004\035\004\030\004\025}{}% 75
\BOOKMARK [0][-]{chapter*.37}{\376\377\004\041\004\037\004\030\004\041\004\036\004\032\000\040\004\030\004\041\004\037\004\036\004\033\004\054\004\027\004\036\004\022\004\020\004\035\004\035\004\053\004\045\000\040\004\030\004\041\004\042\004\036\004\047\004\035\004\030\004\032\004\036\004\022}{}% 76
\BOOKMARK [0][-]{appendix.A}{\376\377\004\030\004\075\004\102\004\065\004\100\004\104\004\065\004\071\004\101}{}% 77
\BOOKMARK [0][-]{appendix.B}{\376\377\004\036\004\102\004\107\004\121\004\102\004\113\000\040\004\076\000\040\004\100\004\060\004\061\004\076\004\102\004\065\000\040\004\101\004\065\004\100\004\062\004\070\004\101\004\060}{}% 78
